<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TcpClient</class>
 <widget class="QWidget" name="TcpClient">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>464</width>
    <height>280</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>网盘客户端</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="name_lb">
         <property name="font">
          <font>
           <family>Adobe Devanagari</family>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="text">
          <string>用户名：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="name_le">
         <property name="font">
          <font>
           <family>Adobe Devanagari</family>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="pwd_lb">
         <property name="font">
          <font>
           <family>Adobe Devanagari</family>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="text">
          <string>密  码：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="pwd_le">
         <property name="font">
          <font>
           <family>Adobe Devanagari</family>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="login_pb">
       <property name="font">
        <font>
         <family>Adobe Devanagari</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>登录</string>
       </property>
       <property name="icon">
        <iconset resource="icon.qrc">
         <normaloff>:/icon/login.png</normaloff>:/icon/login.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="regist_pb">
         <property name="font">
          <font>
           <family>Adobe Devanagari</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="toolTip">
          <string/>
         </property>
         <property name="toolTipDuration">
          <number>-1</number>
         </property>
         <property name="text">
          <string>注册</string>
         </property>
         <property name="icon">
          <iconset resource="icon.qrc">
           <normaloff>:/icon/regist.png</normaloff>:/icon/regist.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="logout_pb">
         <property name="font">
          <font>
           <family>Adobe Devanagari</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>注销</string>
         </property>
         <property name="icon">
          <iconset resource="icon.qrc">
           <normaloff>:/icon/logout.png</normaloff>:/icon/logout.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="icon.qrc"/>
 </resources>
 <connections/>
</ui>
